### PR TITLE
Scaffold Ionic app

### PR DESCRIPTION
**Pull request by isabellelii**
#### Description
Changes proposed in this pull request:

* Scaffold ionic app to project
* Cleaned up unnecessary code
*  Deleted menu toggle button from home page

#### What I have learned working on this feature: 
* How to scaffold an ionic application

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+  <!-- The polyfills js is generated during the build process -->
+  <script src="build/polyfills.js"></script>
+
+  <!-- The vendor js is generated during the build process
+       It contains all of the dependencies in node_modules -->
+  <script src="build/vendor.js"></script>
+
+  <!-- The main bundle js is generated during the build process -->
+  <script src="build/main.js"></script>
```
Comment (AKidd95) [inline src/index.html]: Clean up this file please

**Review by AKidd95 — CHANGES_REQUESTED** [review]
maybe clean up the comments in some of the scss files also

File: src/app/app.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+// styles are for the entire app and not just one component.
+// Additionally, this file can be also used as an entry point
+// to import other Sass files to be included in the output CSS.
+//
+// Shared Sass variables, which can be used to adjust Ionic's
+// default Sass variables, belong in "theme/variables.scss".
+//
+// To declare rules for a specific mode, create a child rule
+// for the .md, .ios, or .wp mode classes. The mode class is
+// automatically applied to the <body> element in the app.
```
Comment (AKidd95) [inline src/app/app.scss:16]: clean up this one specifically, other than that it looks good to me

Comment (isabellelii) [reply src/app/app.scss:16]: We will leave it for now, we are not using this file so it's not relevant no clean this up

**Review by AKidd95 — APPROVED** [review]

**Pull request merged**